src/generic/generic-component-page-header/generic-component-page-header.twig

```diff
@@ -1,67 +1,74 @@
 {#
-  Accepts the following context variables:
-  - variant: (string) Could be: default, basic, highlight
+  Parameters:
+    - "identity" (string) (default: ''): name of the site, i.e. "Digital single market"
+    - "breadcrumb" (array)  (default: []): collection of @ecl/generic-component-link
```

Comment:
it's `@ecl/generic-component-breadcrumb`

Reply:
nope, it's `@ecl/generic-component-link` :) because the breadcrumb template is included like this: `'segments': _breadcrumb,` 

renaming the parameter could make sense

Reply:
You're right, actually it's ok. It's simple, my confusion came from the fact i'm focused on the contents of the template itself.